sales.html

```diff
@@ -6,17 +6,8 @@
     <link rel="stylesheet" href="style.css">
   </head>
   <body>
-    <section>
-      <ul id="firstAndPike-projections">
-      </ul>
-      <ul id="seatacAirport-projections">
-      </ul>
-      <ul id="seattleCenter-projections">
-      </ul>
-      <ul id="capitolHill-projections">
-      </ul>
-      <ul id="alki-projections">
-      </ul>
+    <section id="cookieSalesTable">
```

Comment:
This is super nitpicky, but try to use kebab case or snake case for ids and classes and reserve camel case for javascript variables.